### DISCUSSION TITLE
Invalid Access Token Exception

### DISCUSSION THREAD

**Discussion by ankitbhansali** [egovernments/Digit-Core#130] [Bugs]
Currently, All pods are running fine in QUick Setup. 
I can access the employee page.
But when i run the postman collection to create super user, i get the below response.

"Errors": [
        {
            "code": "BadRequest",
            "message": "400 : [{\"ResponseInfo\":null,\"Errors\":[{\"code\":\"InvalidAccessTokenException\",\"message\":\"An unhandled exception occurred on the server\",\"description\":\"Invalid Access Token Exception\",\"params\":null}]}]",
            "description": "400 : [{\"ResponseInfo\":null,\"Errors\":[{\"code\":\"InvalidAccessTokenException\",\"message\":\"An unhandled exception occurred on the server\",\"description\":\"Invalid Access Token Exception\",\"params\":null}]}]",
            "params": null
        }
    ]

**Comment by talele08** [comment]
You need to port-forward to create the root user. Please use the command:
kubectl port-forward USER_SERVICE_PODNAME 8080:8080

You can then call the API with host: localhost:8080

**Comment by talele08** [comment]
Duplicate of: https://github.com/egovernments/Digit-Core/discussions/131
